spinel-cli.py

```diff
@@ -2422,6 +2426,17 @@ def main():
     if options.debug:
         CONFIG.debug_set_level(options.debug)
 
+    # Obtain the vendor module path, if provided
+    if not options.vendor_path:
+        options.vendor_path = os.environ.get("SPINEL_VENDOR_PATH")
+
+    if options.vendor_path:
+        options.vendor_path = os.path.abspath(options.vendor_path)
+        vendor_path, vendor_module = os.path.split(options.vendor_path)
+        sys.path.insert(0, vendor_path)
```

Comment:
I discussed this briefly with @rlubos today. I think this is more than warranted in this particular case, in order to solve this problem without extra complications and with very little risk. That said, I'd very much like @mbolivar-nordic to confirm this view.

Reply:
> That said, I'd very much like @mbolivar-nordic to confirm this view.

Yep, I think that's reasonable if you want the vendor module to be able to import other modules next to it.